docs/datasets/PublishingAdvanced.md

```diff
@@ -7,10 +7,13 @@ The previously described options to [publish datasets](Publishing.md) in SciCat
 This diagram shows the essential steps in the workflow to be implemented. Please note, that SciCat datasets are *always* only *meta datasets*, SciCat has no direct to the storage system there is no default coupling to such systems.
 
 ### 1. Create a list of selected datasets
-User can select datasets to create a **dataset list**; more datasets can be added and removed in several sessions. He can cancel the process at any time. New will be that while examining single datasets he can directly add or remove them to or from the selection in the cart.
+User can select datasets to create a **dataset list**; more datasets can be added and removed in several sessions. He can cancel the process at any time. New will be that while examining single datasets he can directly add or remove them to or from the selection in the cart. Before proceeding, the user will be asked to verify the selection of datasets. The user has finalized the dataset selection for which he wants to minted a DOI.
+
+##### Internal review (to be implemented)
+Some institutions may introduce an internal review step at this point: other authenticated user (as part of a dedicated reviewer-group) review the selected datasets. If OK, proceed to next form and the initial user can continue the minting process.
 
 ### 2. Fill the form for this dataset selection 
-The user will be forwarded to a form where one can **provide metadata specific to this selection** which can e.g. match site specific information about e.g. grants, associated projects, etc. All selected datasets will be made public. You will be asked to verify the selection of datasets. Owners and Admins are allowed to update this form. Again this shall be possible within several sessions.
+The user will be forwarded to a form where he/she provides **metadata specific to this selection** already conform to DataCite metadata fields to match site specific information about e.g. grants, associated projects, etc. All selected datasets will be made public. Owners and Admins are allowed to update this form. Again this shall be possible within several sessions.
```

Comment:
again you could write in second person here. Also you can say “they” here now or “s/he” if you want to have brevity